R/derive_param_tte.R

```diff
@@ -669,32 +669,6 @@ extend_source_datasets <- function(source_datasets,
 #' @seealso [derive_param_tte()], [censor_source()], [event_source()]
 #'
 #' @return An object of class `tte_source`
```

Comment:
How come we are taking out the example?

Reply:
They didn't like un-exported functions having examples where you make use of `:::`.